python_modules/dagster/dagster_tests/core_tests/snap_tests/snapshots/snap_test_pipeline_snap.py

```diff
@@ -1149,13 +1149,13 @@
     ]
   },
   "tags": {}
-}'''
+}"""
 
-snapshots['test_basic_dep_fan_out 2'] = '9028d026f4732a6e5bc7e86f99c5fdee35619abc'
+snapshots["test_basic_dep_fan_out 2"] = "9028d026f4732a6e5bc7e86f99c5fdee35619abc"
```

Comment:
do all these snapshot updates make sense? they have new Map types in the mix?

Reply:
These are my autoformatter's fault I think... let me regenerate them and avoid opening the file

Reply:
(the contents of most of these are unchanged)

Reply:
Fixed the quote changes. For some reason when I regenerate the snapshots the newlines get removed though. Unfortunate.

Reply:
ah my bad should have caught id match